dotcom-rendering/src/palette.ts

```diff
@@ -6352,6 +6352,10 @@ const paletteColours = {
 		light: matchActiveTabBorderLight,
 		dark: matchActiveTabBorderDark,
 	},
+	'--most-viewed-description': {
```

Comment:
All other properties in this file look to be extracted out. Could we do the same here for consistency purposes?

Reply:
There are [10 other instances of this pattern](https://github.com/guardian/dotcom-rendering/blob/25727ee84d4d1bc377d0800cd6e107aa77d1d540/dotcom-rendering/src/palette.ts#L6723-L6745), but if you’d prefer it extracted I see no objection – you’ll look after this codebase longer than I.

Worth doing the entire extraction in a separate, follow-up PR?

Reply:
It's a very, very small thing so I'd suggest leave it until the next person touches this file